marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/data-services/entitySearch/exportSearch.api

```diff
@@ -36,11 +36,16 @@
             "name": "columns",
             "datatype": "string",
             "multiple" : true
+        },
+        {
+            "name": "acceptType",
+            "datatype": "string"
         }
     ],
     "return": {
-        "datatype": "jsonDocument",
-        "$javaClass": "com.fasterxml.jackson.databind.JsonNode",
-        "description": "Returns a json representation (AST) of an optic plan."
+        "datatype": "textDocument",
+        "$javaClass": "java.io.Reader",
+        "description": "Returns the results of an optic plan",
```

Comment:
Is one Reader returned for every row? If so, I think this description should make that clear (the fact that an optic plan is used is an implementation detail) - e.g. "Returns an array of Reader objects, one for each search result represented as CSV"

Reply:
It only does one reader for the CSV, but the signature allows for multiple, should we want to export them as separate JSON documents for example.

Reply:
Although I understand the intention behind this, I feel like this creates some confusion wrt the returned reader and its relationship with the CSV. For the sake of simplicity should we change the signature to represent what's currently being returned?

Reply:
I feel endpoints are cheap, and reuse can always be done via library modules. So I lean towards fine-grained, explicitly-named endpoints - e.g. "exportSearchResultsAsCsv" - that reuse libraries to avoid duplication. We can always create more generic endpoints in the future as we learn more.